docs/multicluster/user-guide.md

```diff
@@ -592,43 +633,105 @@ In future releases, some additional tooling may become available to automate the
 creation of ResourceExports for ACNPs, and provide a user-friendly way to define
 Multi-cluster NetworkPolicies to be enforced in the ClusterSet.
 
-## Multi-cluster Pod-to-Pod Connectivity
+## Multi-cluster Stretched NetworkPolicy
 
-Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters
-through Multi-cluster Gateways. Pod IPs can be reached in all member clusters
-within a ClusterSet. To enable this feature, the cluster's Pod CIDRs must be set in
-ConfigMap `antrea-mc-controller-config` of each member cluster like the example
-below. Note, **Pod CIDRs must not overlap among clusters to enable cross-cluster
-Pod-to-Pod connectivity**.
+Antrea-native policies can be enforced on cross-cluster traffic in a ClusterSet in the following ways:
+
+### Egress NetworkPolicy to Multi-cluster Services
+
+Restricting Pod egress traffic to backends of a Multi-cluster Service (which can be on the
+same cluster of the source Pod or on a different cluster) is supported by Antrea-native
+policy's `toMulticlusterServices` feature in egress rules. To define such a policy, simply
```

Comment:
We should mention this is supported from Antrea 1.10, and before that use `toServices` with the imported Service name instead.